kong/plugins/httplog/log.lua

```diff
@@ -0,0 +1,19 @@
+local cjson = require "cjson"
+local http_client = require "kong.tools.http_client"
+
+local APPLICATION_JSON_TYPE = "application/json"
+
+local _M = {}
+
+local function log(premature, conf, message)
+  local response, status, headers = http_client.post(conf.logging_url, cjson.encode(message), { ["content-type"] = APPLICATION_JSON_TYPE })
```

Comment:
While I agree on the first comment, I disagree on the second one: any other method other than `POST` is not going to be the best option for this kind of functionality.